app/frontend/components/pages/login/mnemonicAuth.tsx

```diff
@@ -81,7 +83,7 @@ class LoadByMnemonicSectionClass extends Component<Props> {
           placeholder="Enter your wallet mnemonic"
           value={mnemonicInputValue}
           onInput={updateMnemonic}
-          onBlur={checkForMnemonicValidationError}
+          onBlur={updateMnemonicValidationError}
```

Comment:
is sanitizedMnemonic validated in two different places now thanks to 'canSubmit'?

Reply:
Yes. I am open for discussion here but basically, rationale is
- updateValidation() is called whenever we need to display validation things to user
- real validation state can be different and should always be actual.

When thinking about it, maybe I can keep explicit canSubmit flag in state as well. But this would need re-writing the state shape into substate...